examples/shader/shader_material.rs

```diff
@@ -91,6 +95,16 @@ impl RenderAsset for CustomMaterial {
 }
 
 impl Material for CustomMaterial {
+    // When creating a custom material, you need to define either a vertex shader, a fragment shader or both.
+    // If you don't define one of them it will use the default mesh shader which can be found at
+    // <https://github.com/bevyengine/bevy/blob/latest/crates/bevy_pbr/src/render/mesh.wgsl>
+
+    // For this example we don't need a vertex shader
+    // fn vertex_shader(asset_server: &AssetServer) -> Option<Handle<Shader>> {
```

Comment:
Can we put this function in a code block?

Reply:
Do those actually work in non-doc comments? 🤔

Reply:
I'm not sure, but that code block is not just an example, it would actually work if it was uncommented which is why I didn't put it in a doc comment.